src/stores/DOM_focus_store.ts

```diff
@@ -4,5 +4,6 @@ export class DOMFocusableElementStore {
 
   setFocusableElement(element: HTMLElement) {
```

Comment:
didn't you want to make the `focusableElement` private again ?

Reply:
indeed ><